docs/code_review.md

```diff
@@ -3,6 +3,11 @@
 **Lets use the following guidelines for Code Reviews and contribution:**
 
 * Create own fork of the repository and submit PR from your fork.
+* If a github issue describing the changes you are making does not exist be sure to
+    create one before creating a pull request. This is a *requirement* for all
+    bugs, feature requests, and enhancements to the framework or our tests that
```

Comment:
I would say feature request can exist in Trello or Jira(I can still copy/paste them in github/issues before PR) but want to bring it up.

Reply:
Yeah that's a good point. As long as those who are expected to review PRs have access to the trello/jira boards I think that's fine.